test/xpu_api/utilities/optional/optional.object/optional.object.ctor/deduct.pass.cpp

```diff
@@ -0,0 +1,82 @@
+// -*- C++ -*-
```

Comment:
The test name is confusing. Consider `optional_deduction_guides.pass.cpp` instead.

Reply:
Fixed